assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -42,6 +42,30 @@ spec:
 {{ if .MasterPriorityClass }}
       priorityClassName: {{ .MasterPriorityClass }}
 {{ end }}
+      initContainers:
+        - name: prepare-payload
+          image: {{ .ReleaseImage }}
+          imagePullPolicy: Always
+          command:
+            - /bin/bash
+          args:
+            - -c
+            - -|
+              cp -R /manifests /var/payload/
+              cp -R /release-manifests /var/payload/
+
+              # Remove cluster cloud controller manager operator manifests
+              rm /var/payload/release-manifests/0000_26_cloud-controller-manager-operator_00_namespace.yaml
```

Comment:
Is this really the only way to do this?  Is there not a config option to disable?  Also, should this be in the manifest bootstrapper?  And finally, I think we could delete `0000_26_cloud-controller-manager-operator_*` rather than each file by name.

Reply:
The other option is to modify the 3CMO itself by adding an annotation to signal that it should be excluded. You can see an example [here](https://github.com/openshift/cluster-kube-apiserver-operator/blob/c4ccb76e8362e7f97c07fdcd901bcfbbdad8aa37/manifests/0000_20_kube-apiserver-operator_06_deployment.yaml#L11) for the kube-apiserver. We can go that route, but it will require getting that change merged and waiting for a new release to get the change.

Reply:
The way this has worked up to now is that we've used an annotation to exclude certain manifests from being applied by the cvo. Like [this](https://github.com/openshift/cluster-kube-apiserver-operator/blob/c4ccb76e8362e7f97c07fdcd901bcfbbdad8aa37/manifests/0000_20_kube-apiserver-operator_06_deployment.yaml#L11) However, since the introduction of [cluster profiles](https://github.com/openshift/enhancements/pull/200) we've been trying to switch to an include annotation. Like [this](https://github.com/openshift/cluster-kube-apiserver-operator/blob/c4ccb76e8362e7f97c07fdcd901bcfbbdad8aa37/manifests/0000_20_kube-apiserver-operator_00_namespace.yaml#L5). Several other changes were needed to make that work properly and now we're almost there. Switching to using the include annotation vs the exclude annotation is simply a matter of changing environment variables in the cluster version operator's deployment like [this](https://github.com/openshift/hypershift/pull/539/files#diff-520b6ecfad21e6c9bc0cbf244ff694cf5296ffc8c0318cb2248eb7185a36cd8aL152-L153).
It may be good to try switching to use cluster profiles for 4.9 in ibm-roks-toolkit and see if things work.

Reply:
PR has been updated to switch over to the `ibm-cloud-managed` profile